index.html

```diff
@@ -4,13 +4,152 @@
+      <div class="home-intro-location-container">
+        <h2>2023.05.24(WED) ~ 26(FRI)</h2>
+        <p>@ Chapelco Golf Resort, San Martín de Los Andes, Argentine Patagonia. </p>
+      </div>
+    </div>
+    </section>
+
+    <section id="program" class="main-program">
+      <h2>Main Program</h2>
+      <div class="separator"></div>
+
+      <div class="all-activities-container">
+        <div class="activity-container">
+          <div class="activity">
+          <div class="icon-container">
+            <img class="program-icon" src="./Images/Icons/planet-earth-svgrepo-com.svg" alt=""/>
+            <span>Sponsors</span>
+          </div>
+          <p class="activity-paragraph">Know what enviromental care companies are doing about restoration.</p>
+        </div>
+      </div>  
+
+      <div class="activity-container">
+        <div class="activity">
+          <div>
+            <img class="program-icon" src="./Images/Icons/document-envelope-envelope-general-svgrepo-com.svg" alt=""/>
+            <span>Projects</span>
+          </div>
+          <p>Appreciate various proteccionism parks and landscapes recovered using private companies profits.</p>
+        </div>
+      </div>  
+
+      <div class="activity-container">
+        <div class="activity">
+          <div>
+            <img class="program-icon" src="./Images/Icons/average-image-image-images-svgrepo-com.svg" alt=""/>
+            <span>Brand Value</span>
+          </div>
+          <p>Learn about how these iniciatives generate growth in their brands value.</p>
+        </div>
+      </div>  
+
+      <div class="activitiy-container">
+        <div class="activity">
+          <div>
+            <img class="program-icon" src="./Images/Icons/general-graph-increase-svgrepo-com.svg" alt=""/>
+            <span>Tax Impact</span>
+          </div>
+          <p>Learn from Ernst & Young tax experts how to apply a tax program and increase your long term return.</p>
+        </div>
+      </div>  
+
+      <div class="activity-container">
+        <div class="activity">
+          <div>
+            <img class="program-icon" src="./Images/Icons/award-champion-general-svgrepo-com.svg" alt=""/>
+            <span>Leading Case</span>
+          </div>
+          <p>Hear actual companies success cases directly from their leaders.</p>
+        </div>
+      </div>
+
+      </div>
+
+      <button class="join-global-forum mobile">Join Enviromental Forum 2023</button>
+      <a href="#" class="see-whole-program-button desktop">SEE THE WHOLE PROGRAM</a>
+    </section>
+
+  <!-- ------------- Featured Speakers --------------- -->
+
+    <section class="section-speakers">
+      <h2>Invited Speakers</h2>
+      <div class="separator"></div>
+    
+      <div class="dynamic-speakers"> 
+      </div>
+    </section>
+  </main>
+
+  <!-- --------------------- <SPONSORS SECTION> --------------------- -->
+ 
+  <footer class="">
+    <div class="sponsor-section-container desktop">
```

Comment:
Please consider implementing the following to enhance the sponsors section design following design guidelines👇💪
- [ ] I think it would be better to follow the colors given in design guidelines for sponsors
- [ ] Reduce the `font-size` of sponsor's title. Maybe you can try `1.5rem` 🤔
- [ ] What do you think o fusing a `hx` tag instead of `p` for the title? 💭
- [ ] Reduce the space/padding/margin between image and increase around them
- [ ] Can you please add the sponsors section in `main` tag and left the `footer` tag only for legal right? 💪

Your design | Given design 
---| ---|
![image](https://github.com/HFG43/Capstone1/assets/106119105/8f5600fe-bacc-4876-b201-dfec52bd96c0) | ![image](https://github.com/HFG43/Capstone1/assets/106119105/e7389409-055a-4151-8371-4d9f21b64531)